contributors/devel/development.md

```diff
@@ -136,8 +136,8 @@ platform.
 
 ### jq
 
-Kube-apiserver requires the `jq` to be installed to successfully build your Local Kubernetes Deployment. 
-You can see installation guide for `jq` over [here](https://stedolan.github.io/jq/download/).
+Kube-apiserver requires `jq` to be installed to successfully build your local Kubernetes environment. 
+You can see the installation guide for `jq` [here](https://stedolan.github.io/jq/download/).
```

Comment:
suggest: The [`jq` installation guide]()] provides detailed instructions for supported platforms.

Reply:
Good suggestion, updated.